receiver/webhookeventreceiver/config.go

```diff
@@ -29,6 +29,7 @@ type Config struct {
 	HealthPath                 string                   `mapstructure:"health_path"`                   // path for health check api. Default is /health_check
 	RequiredHeader             RequiredHeader           `mapstructure:"required_header"`               // optional setting to set a required header for all requests to have
 	SplitLogsAtNewLine         bool                     `mapstructure:"split_logs_at_newline"`         // optional setting to split logs into multiple log records
+	SplitLogsAtJSONBoundary    bool                     `mapstructure:"split_logs_at_json_boundary"`   // optional setting to split logs at JSON object boundaries
```

Comment:
kindof a nit but since we're adding this as an optional setting can you explicitly set its default value to false in the `factory.go`?